guides.md

```diff
@@ -0,0 +1,83 @@
+# Guides
+
+Guides explain in practical terms how we do stuff. Any Hypha member can
+change these.<sup>[1](#references)</sup>
+
+We are inspired by the idea of "protocols" from the [Lab
+Book][clear-lab-book] of the [Civic Laboratory for Environmental Action
+Research (CLEAR)][clear]
+
+> Protocols are the "attitudes" or "the manner in which one approaches each and
+every element in our space." They are different than pure rules or
+instructions; they are ways that we establish order and maintain practices
+across our group [...] -- They define the way we ought to proceed or behave in
+different situations. As such, they are normative, or premised on values,
+morals, and an idea of how things ought to be done. They are a manifestation of
+our values.
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+members. Skip below steps at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic, and ask
+  who's interested to participate. Allow a day for people to respond.
+- Consider who might be most interested, and what timing might work for
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
+  - Check everyone's [weekly availability `schedules`][schedules].
+- If 2-3 candidate timeslots feel obvious from looking at internal
+  availabilities:
```

Comment:
clarify what "internal availabilities" means (I'm really struggling), could we say "people's availability"?

Reply:
"internal availabilities" was just reference to one standing internal when2meet for members. i'm just trying to find language to avoid saying "look at when2meet"... which itself can be a "fresh" one, or using that standing internal one. As in, when scheduling with Jutta, I didn't blow open the scheduling to include everyone, but I checked our "internal availability" on that standing when2meet.

Is this worth capturing here? Happy to use whatever lingo, or even to just drop it as TMI :)

Reply:
Maybe these could use clearer lingo...

- "availability calendar" (gcal) vs "availability schedule" (when2meet)?

Reply:
dc and i spoke and sorted 🌮 #goodenoughtada